javascript/javascript3/week3/index.js

```diff
@@ -0,0 +1,92 @@
+// Fetch movies from this api:
+// Create an array of bad movies
+// Creat an array of bad movies since year 2000
+fetch(
+    "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+  )
+  .then((moviesResponse) => moviesResponse.json())
+  .then(data => {
+    const getBadMovies = data.filter(m => m.rating < 6)
```

Comment:
I would use some descriptive name instead of m.

Reply:
Thank you for your feedback 🙂